libqtile/layout/xmonad.py

```diff
@@ -138,9 +138,9 @@ class MonadTall(Layout):
 
     """
 
-    _left = 0
-    _right = 1
-    _min_height = 85
+    _align_min = 0
+    _align_max = 1
+    _min_secondary_size = 85
     _min_ratio = .25
     _med_ratio = .5
     _max_ratio = .75
```

Comment:
Ah, well, there's no guarantee everything in the current code base is a good idea :)